dpe/src/bitmap.rs

```diff
@@ -0,0 +1,78 @@
+/*++
+Licensed under the Apache-2.0 license.
+
+Abstract:
+    Helper struct for controlling and viewing bitmaps.
+--*/
+
+#[repr(C)]
+#[derive(Debug, PartialEq, Eq, Default)]
```

Comment:
Do we want to make this Copy? If yes, we may want to take `self` instead of `&self` in methods.